### PR TITLE
Feature/332 create topside asset

### PR DESCRIPTION
**Pull request by ManveerPandher**
Reason for case:
- When creating a project, there should be possible to assign a current or existing "Topside", aswell as edit the content.

What has been done:
- Replicated the way Substructure Asset was created with its own View, and added existing fields to datamodels within Frontend and Backend

### CODE REVIEW

File: frontend/src/Views/TopsideView.tsx [hunk 1]
```diff
@@ -1,113 +1,182 @@
...
+    const onCellsChanged = (changes: { cell: { value: number }; col: number; row: number; value: string }[]) => {
         const newGridData = replaceOldData(gridData, changes)
         setGridData(newGridData)
-        setColumns(columnTitles)
+        setColumns(getColumnAbsoluteYears(caseItem, topside?.topsideCostProfile))
+        setHasChanges(true)
+    }
+
+    const onImport = (input: string, year: number) => {
+        console.log("Before copying")
```
Comment (petterwildhagen) [inline frontend/src/Views/TopsideView.tsx:113]: Delete console log debug statements?

**Review by petterwildhagen — CHANGES_REQUESTED** [review]
Looks good, just delete the console log statements and I will approve

File: frontend/src/Views/TopsideView.tsx (hunk 1 above)
Comment (ManveerPandher) [reply frontend/src/Views/TopsideView.tsx]: @petterwildhagen 
I've deleted the console log debug statemtents

**Review by petterwildhagen — APPROVED** [review]
Looks good

**Review by petterwildhagen — APPROVED** [review]
Looks good!!!

**Pull request merged**